lib/cn/spill.c

```diff
@@ -276,6 +274,10 @@ cn_subspill(
 
         w->cw_stats.ms_keys_out++;
         w->cw_stats.ms_key_bytes_out += key_obj_len(&sctx->pt_kobj);
+
+        ss->ss_added = true;
+        if (key_obj_cmp_prefix(&sctx->pt_kobj, &ekobj) < 0)
+            sctx->pt_set = false;
```

Comment:
This patch is a ptomb fix from @gsramdasi , Gaurav, do you want me to include it here, or do you want to run your patch separately through the gauntlet?

Reply:
Go for it!  I will prob have to remerge to nabeel's PR after that goes in...